File: back-end/src/main/java/com/spring/restaurant/model/Order.java
package com.spring.restaurant.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "orders")
public class Order extends CategoryOrder {
    @Column(name = "price")
    private int price;
    @Column(name = "image")
    private String img;

    @Column(name = "description",length = 2000)
    private String description;

    @ManyToOne
    @JoinColumn(name = "id_category")
    private Category category;

}

File: frontend/src/model/order.ts
export class Order {

  id:number;
  data_update:Date;
  data_create:Date;
  name:string;
  description:string;
  image:string;
  price:number;


constructor(id:number,data_update:Date,data_create:Date,name:string,description:string,image:string,price:number){
  this.id=id;
  this.data_update=data_update;
  this.data_create=data_create;
  this.name=name;
  this.description=description;
  this.image=image;
  this.price=price;
}

}
